include/util/vector_view.hpp

```diff
@@ -143,8 +143,33 @@ template <> class vector_view<bool>
     unsigned *m_ptr;
     std::size_t m_size;
 
+    static constexpr std::size_t UNSIGNED_BITS = CHAR_BIT * sizeof(unsigned);
+
   public:
     using value_type = bool;
+    struct reference
```

Comment:
Nitpick/Questions: I am not entirely sure on the substructures naming convention, so feel free to ignore: shouldn't this be capital case?